cime_config/testdefs/testmods_dirs/clm/FatesHydro/user_nl_clm

```diff
@@ -1,5 +1,9 @@
 hist_empty_htapes = .true.
 use_fates_planthydro= .true.
-hist_fincl1       = 'FATES_ERRH2O_SCPF','FATES_TRAN_SCPF','FATES_ROOTUPTAKE_SCPF','FATES_ROOTUPTAKE01_SCPF',
-'FATES_SAPFLOW_COL_SCPF','FATES_ITERH1_COL_SCPF','FATES_ATH_COL_SCPF','FATES_TTH_COL_SCPF',
-'FATES_STH_COL_SCPF','FATES_LTH_COL_SCPF','FATES_AWP_COL_SCPF','FATES_BTRAN_COL_SCPF'
+hist_fincl1       = 'FATES_ERRH2O_SCPF','FATES_TRAN_SCPF',
```

Comment:
Looks like this is just a subset of the total history variables added in the fates PR yes?  Just want to confirm.

Reply:
yeah, this test focuses just on the hydraulics diagnostic variables.  All of the new ones are in here.